site/src/components/Dashboard/DeploymentBanner/DeploymentBannerView.tsx

```diff
@@ -40,44 +39,23 @@ import { getDisplayWorkspaceStatus } from "utils/workspace";
 import { colors } from "theme/colors";
 import { HelpTooltipTitle } from "components/HelpTooltip/HelpTooltip";
 import { Stack } from "components/Stack/Stack";
+import { type ClassName, useClassName } from "hooks/useClassName";
 
 export const bannerHeight = 36;
 
-const styles = {
-  group: css`
-    display: flex;
-    align-items: center;
-  `,
-  category: (theme) => ({
-    marginRight: 16,
-    color: theme.palette.text.primary,
-  }),
-  values: (theme) => ({
-    display: "flex",
-    gap: 8,
-    color: theme.palette.text.secondary,
-  }),
-  value: css`
-    display: flex;
-    align-items: center;
-    gap: 4px;
-
-    & svg {
-      width: 12px;
-      height: 12px;
-    }
-  `,
-} satisfies Record<string, Interpolation<Theme>>;
-
 export interface DeploymentBannerViewProps {
   health?: HealthcheckReport;
   stats?: DeploymentStats;
   fetchStats?: () => void;
 }
 
-export const DeploymentBannerView: FC<DeploymentBannerViewProps> = (props) => {
-  const { health, stats, fetchStats } = props;
+export const DeploymentBannerView: FC<DeploymentBannerViewProps> = ({
+  health,
+  stats,
+  fetchStats,
+}) => {
```

Comment:
I thought you wanted to destructure props in the component body? JC about the change; I don't mind either way.

Reply:
I don't mind it so much when using the `FC` type, because the props type is still easily visible at the top, which is really the reason I don't like inline destructuring: it pushes the type after the destructure way down and makes it harder to spot.

I think if other peeps line inline destructuring, this could be a nice compromise